pyvrp/plotting/plot_objectives.py

```diff
@@ -50,9 +50,10 @@ def _plot(x, y, *args, **kwargs):
     y = [d.avg_cost for d in result.stats.feas_stats]
     _plot(x, y, label="Feas. avg.", c="tab:green", alpha=0.3, ls="--")
 
-    # Use best objectives to set reasonable y-limits
-    best = np.nanmin([d.best_cost for d in result.stats.feas_stats])
-    ax.set_ylim(best * ylim_adjust[0], best * ylim_adjust[1])
+    # Use best objectives to set reasonable y-limits, if available.
+    best = np.min([d.best_cost for d in result.stats.feas_stats])
+    if not np.isnan(best):
+        ax.set_ylim(best * ylim_adjust[0], best * ylim_adjust[1])
```

Comment:
I don't think works as intended. `results.stats.feas_stats` often has NaNs because the first few iterations do not produce feasible solutions. So using `np.min` returns `nan` in most cases and the y-limits won't be set. That also seems to be the reason why the test figure changed (which isn't supposed to change).

I think we have to filter on NaNs first. `np.nanmin` doesn't work on all-NaN arrays so something like this should work:

```python
best_costs = [d.best_cost for d in result.stats.feas_stats() if not np.isnan(d.best_cost)]
if best_costs:
	best = min(best_costs)
	ax.set_ylim(best * ylim_adjust[0], best * ylim_adjust[1])
```

Reply:
I think this simplifies further to just:
```python
    if result.is_feasible():
        best_cost = result.cost()
        ax.set_ylim(best_cost * ylim_adjust[0], best_cost * ylim_adjust[1])
```